tree-sitter-verilog/build.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+set -x
+
+npm config set python python2.7
```

Comment:
Why is this setting python2.7?

Reply:
Generating `tree-sitter-verilog.so` requires both `python2` and `python3`, `python2` is used by `node-gyp` during `npm install`, when it is not specified like that, it tries to use `python3` and fails on `print` without parentheses during `node-gyp` installation

Reply:
Can you log a bug against upstream for this?

Reply:
Hrm

https://github.com/nodejs/node-gyp/blob/master/README.md#installation
> NOTE: node-gyp is compatible with Python v2.7, v3.5, v3.6, or v3.7. If the Python to use is not explicitly configured (see "Configuring Python Dependency" below) it will attempt to find a compatible Python executable.

Reply:
I noticed now that somehow old version of `node-gyp` is used during build: `gyp ERR! node-gyp -v v3.8.0` and on a branch for that [version](https://github.com/nodejs/node-gyp/blob/v3.x/README.md#on-unix) it says:
> python (v2.7 recommended, v3.x.x is not supported)

I will check if it is possible to to use the new version which supports python3

Reply:
It turns out that `node-gyp` comes [bundled](https://github.com/nodejs/node-gyp/wiki/Updating-npm%27s-bundled-node-gyp) with `npm` and `npm` from conda repository comes with version `3.8.0` of `node-gyp` so updating to `conda-forge` version of `nodejs` should help with that.

At the same time I noticed that we don't need to run `npm install` but only `npm install tree-sitter-cli` and after that `./node_modules/tree-sitter-cli/tree-sitter generate` which is both faster and doesn't use `node-gyp` at all.